### ISSUE TITLE
Keyword Search always return score 0.0 and seems happen after v1.24

### ISSUE THREAD

**Issue by kun432** [weaviate/weaviate#4504]
### How to reproduce this bug?

create a collection:

```python
from datasets import load_dataset
import weaviate
import weaviate.classes as wvc

dataset = load_dataset('jeopardy', split='train').shuffle(seed=42)
sample_dataset = dataset.select(range(500))
sample_data = [{col: row[col] for col in ["category", "question", "answer"]} for row in sample_dataset]

client = weaviate.connect_to_embedded(
    version="1.24.4"
)

questions = client.collections.create(
    name="Question",
)

question_objs = list()
for i, d in enumerate(sample_data):
    question_objs.append({
        "answer": d["answer"],
        "question": d["question"],
        "category": d["category"],
    })

questions.data.insert_many(question_objs)
```

search:

```python
response = questions.query.bm25(
    query="america",
    query_properties=["question","answer","category"],
    return_metadata=wvc.query.MetadataQuery(score=True, explain_score=True),
    limit=10
)

for r in response.objects:
    print(r.metadata.score)
    print(r.metadata.explain_score)
    print(r.properties)
    print()
```

result:

```text
0.0
, BM25F_america_frequency:1, BM25F_america_propLength:2
{'answer': 'Jean Lafitte', 'question': "'A Natl. Historical Park & Preserve named for this pirate includes the site of the Battle of New Orleans'", 'category': 'HISTORIC AMERICA'}

0.0
, BM25F_america_propLength:3, BM25F_america_frequency:1
{'answer': '"God Bless America"', 'question': '\'This Irving Berlin song has been called "The Nation\'s Unofficial Second National Anthem"\'', 'category': 'IRVING BERLIN'}

0.0
, BM25F_america_frequency:1, BM25F_america_propLength:3
{'answer': 'Rockford', 'question': '\'The "files" on this large Illinois city include its historic leadership in screw production\'', 'category': 'AMERICA THE BEAUTIFUL'}

0.0
, BM25F_america_propLength:17, BM25F_america_frequency:3
{'answer': 'law', 'question': '\'"America!  America!  God mend thine every flaw/ Confirm thy soul in self-control, thy liberty in" this\'', 'category': 'AMERICA THE BEAUTIFUL'}

0.0
, BM25F_america_frequency:1, BM25F_america_propLength:11
{'answer': 'Milwaukee', 'question': '\'This city on Lake Michigan is "The Beer Capital of America"\'', 'category': 'AMERICAN CITIES'}

0.0
, BM25F_america_frequency:1, BM25F_america_propLength:16
{'answer': 'CNN Headline News', 'question': "'Every half hour since 1982, this CNN network has updated America on news, sports, business & entertainment'", 'category': 'CNN'}
```

### What is the expected behavior?

return scores calculated by query relavance.

### What is the actual behavior?

return always score 0.0.

### Supporting information

tried on various versions, but this happens.

- WCS: 1.24.1
- Docker-Compose: 1.24.1
- Embedded: 1.24.1, 1.24.4

But v.1.23.13, (only tried on Embedded), correct scores return.

```python
(snip: same code as above)

client = weaviate.connect_to_embedded(
    version="1.23.13",
)

(snip: same code as above)
```

```
``text
2.748971939086914
, BM25F_america_frequency:1, BM25F_america_propLength:2
{'answer': 'Jean Lafitte', 'question': "'A Natl. Historical Park & Preserve named for this pirate includes the site of the Battle of New Orleans'", 'category': 'HISTORIC AMERICA'}

2.52490496635437
, BM25F_america_frequency:1, BM25F_america_propLength:3
{'answer': '"God Bless America"', 'question': '\'This Irving Berlin song has been called "The Nation\'s Unofficial Second National Anthem"\'', 'category': 'IRVING BERLIN'}

2.52490496635437
, BM25F_america_propLength:3, BM25F_america_frequency:1
{'answer': 'Rockford', 'question': '\'The "files" on this large Illinois city include its historic leadership in screw production\'', 'category': 'AMERICA THE BEAUTIFUL'}

2.293008327484131
, BM25F_america_frequency:3, BM25F_america_propLength:17
{'answer': 'law', 'question': '\'"America!  America!  God mend thine every flaw/ Confirm thy soul in self-control, thy liberty in" this\'', 'category': 'AMERICA THE BEAUTIFUL'}

1.5283229351043701
, BM25F_america_frequency:1, BM25F_america_propLength:11
{'answer': 'Milwaukee', 'question': '\'This city on Lake Michigan is "The Beer Capital of America"\'', 'category': 'AMERICAN CITIES'}

1.225906491279602
, BM25F_america_frequency:1, BM25F_america_propLength:16
{'answer': 'CNN Headline News', 'question': "'Every half hour since 1982, this CNN network has updated America on news, sports, business & entertainment'", 'category': 'CNN'}
```

### Server Version

1.24.x

### Code of Conduct

- [X] I have read and agree to the Weaviate's [Contributor Guide](https://weaviate.io/developers/contributor-guide) and [Code of Conduct](https://weaviate.io/service/code-of-conduct)

**Comment by donomii** [comment]
Thank you very much for your bug report.  I have created the solution here:

https://github.com/weaviate/weaviate/pull/4505

Please note that your search results are still correct (in the right order), however the score was dropped when preparing the output

**Comment by kun432** [comment]
Thanks for super fast fix!

>Please note that your search results are still correct (in the right order), however the score was dropped when preparing the output

I guess so. hybrid search results and its ranking seems woking correctly in my end. very looking forward to be merged and released in v1.24 latest because I want to try Japanese tokenizer! (also tokenizer itself seems working correctly too, but this make us confused.)

**Comment by kun432** [comment]
I confirmed this has been fixed on Embedded v1.24.5.

Thank you so much!

**Comment by donomii** [comment]
Thank you for letting us know.  We hope you enjoy using Weaviate
